edgexfoundry/device-usb-camera
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
1. run edgex non-secure stack
3. build device service in docker- `make docker`
4. Add the device service to the compose file `make gen <previous options> ds-usb-camera`
5. Run `make up` to start the device service in docker
6. Inspect the logs to verify that common configuration is loaded from consul
7. verify that private configuration is loaded in consul
8. ensure that writable in the private config is modifiable - change log level to DEBUG and see change in the logs

## New Dependency Instructions (If applicable)
